mozilla-release/mobile/android/app/mobile.js

```diff
@@ -914,8 +914,8 @@ pref("extensions.systemAddon.update.url", "");
 // override the UA on Google SERP pages due to https://bugzilla.mozilla.org/show_bug.cgi?id=975444
 pref("general.useragent.override.encrypted.google.com", "Mozilla/5.0 (Linux; Android 8.0.0; rv:61.0) FxQuantum/61.0 AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.0.0 Mobile Safari/537.36");
 pref("general.useragent.override.encrypted.google.de", "Mozilla/5.0 (Linux; Android 8.0.0; rv:61.0) FxQuantum/61.0 AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.0.0 Mobile Safari/537.36");
-pref("general.useragent.override.google.com", "Mozilla/5.0 (Linux; Android 8.0.0; rv:61.0) FxQuantum/61.0 AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.0.0 Mobile Safari/537.36");
-pref("general.useragent.override.google.de", "Mozilla/5.0 (Linux; Android 8.0.0; rv:61.0) FxQuantum/61.0 AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.0.0 Mobile Safari/537.36");
+pref("general.useragent.override.www.google.com", "Mozilla/5.0 (Linux; Android 8.0.0; rv:61.0) FxQuantum/61.0 AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.0.0 Mobile Safari/537.36");
+pref("general.useragent.override.www.google.de", "Mozilla/5.0 (Linux; Android 8.0.0; rv:61.0) FxQuantum/61.0 AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.0.0 Mobile Safari/537.36");
```

Comment:
Can we update to a newer UA here too?

Reply:
newer ones do not work, only some particular ones work fine :/ not sure what logic they use to show the nice results